skopeo/README.md

```diff
@@ -38,3 +38,24 @@ podman run docker://quay.io/skopeo/stable:latest copy docker://quay.io/skopeo/st
 # Inspect the fedora:latest image
 podman run docker://quay.io/skopeo/stable:latest inspect --config docker://registry.fedoraproject.org/fedora:latest  | jq
 ```
+
+## Use against a private registry server is possible but somewhat complex
+
+1. Assuming one isn't already defined, setup a podman secret with the `auth.json` contents.
+   See the `containers-auth.json` man page for the format.
```

Comment:
This is the right URL, I'm just not 100% sure about the backtick placement  I have.  I think it's right, but...

```suggestion
   See the [`containers-auth.json`](https://github.com/containers/image/blob/main/docs/containers-auth.json.5.md) man page for the format.
```

Reply:
Nope, that looks correct to me.  Thanks for the suggestion!